sources/ts/index.d.ts

```diff
@@ -1 +0,0 @@
-export {};
```

Comment:
Why did you remove the .d.ts files? They for sure look unneccessary but would love to know how/why you can remove them :)

Reply:
Typescript can handle files without types. I deleted them and everything is fine. It's hard to maintain them on big code base